src/test/java/CommitViewUriBuilderTest.java

```diff
@@ -21,7 +21,7 @@ public void testBuild_AllValid() {
         "intellij",
         "1.1");
 
-    String want = "https://www.sourcegraph.com/github.com/sourcegraph/sourcegraph-jetbrains/-/commit/1fa8d5d6286c24924b55c15ed4d1a0b85ccab4d5?editor=JetBrains&version=v1.2.0&utm_product_name=intellij&utm_product_version=1.1";
+    String want = "https://www.sourcegraph.com/github.com/sourcegraph/sourcegraph-jetbrains.git/-/commit/1fa8d5d6286c24924b55c15ed4d1a0b85ccab4d5?editor=JetBrains&version=v1.2.2&utm_product_name=intellij&utm_product_version=1.1";
```

Comment:
potential regression or bug here - this wouldn't be a valid Sourcegraph URL with `.git` in it.

Reply:
Just checked and the actual feature isn't working with 1.2.2, so likely there is a bug that already existed and just got copied into this test.

Reply:
Updated the issue: https://github.com/sourcegraph/sourcegraph-jetbrains/issues/25#issuecomment-1022512027